novaan-server/src/Auth/AuthService.cs

```diff
@@ -30,12 +30,14 @@ public Task<bool> SignInGoogle()
 
         public async Task<bool> SignUpWithCredentials(SignUpDTO signUpDTO)
         {
-            if (CheckEmailExist(signUpDTO.Email).Result)
+            var emailExisted = await checkEmailExist(signUpDTO.Email);
+            var usernameExisted = await checkUsernameExist(signUpDTO.Username);
```

Comment:
Nên định nghĩa biến từ chỗ nào bắt đầu sử dụng thôi bạn :v
Trừ trường hợp biến sử dụng trong toàn bộ function cần định nghĩa trước để người đọc hiểu context của function

Reply:
oke